scan-payment/src/main/java/com/getbouncer/scan/payment/analyzer/NameDetectAnalyzer.kt

```diff
@@ -0,0 +1,268 @@
+package com.getbouncer.scan.payment.analyzer
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.RectF
+import android.media.MediaScannerConnection
+import android.media.MediaScannerConnection.OnScanCompletedListener
+import android.provider.MediaStore
+import android.util.Log
+import com.getbouncer.scan.framework.Analyzer
+import com.getbouncer.scan.framework.AnalyzerFactory
+import com.getbouncer.scan.framework.ml.ssd.hardNonMaximumSuppression
+import com.getbouncer.scan.payment.ml.AlphabetDetect
+import com.getbouncer.scan.payment.ml.SSDObjectDetect
+import com.getbouncer.scan.payment.ml.SSDOcr
+import com.getbouncer.scan.payment.ml.ssd.DetectionBox
+import com.getbouncer.scan.payment.ml.toObjectDetectionCroppedImage
+import kotlinx.coroutines.CoroutineScope
+import java.util.*
+
+
+private const val NUM_PREDICTION_STRIDES = 10
```

Comment:
can you name this with the unit or add a comment explaining what this is? it looks like this might be pixels of a bitmap or rows of the prediction?